### ISSUE TITLE
[PRO] Warnings on ruby-2.7.1

### ISSUE THREAD

**Issue by michal-samluk** [rmosolgo/graphql-ruby#2937]
graphql-pro is throwing warning for ruby-2.7.1. Here's the stacktrace: 

```
graphql-pro-1.13.4/lib/graphql/pro/relation_connection/column_value_connection.rb:10: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
graphql-1.10.9/lib/graphql/relay/base_connection.rb:61: warning: The called method `initialize' is defined here
```

**Comment by rmosolgo** [comment]
Thanks for the heads up, I'll take a look!

**Comment by rmosolgo** [comment]
I just shipped graphql-pro 1.13.5 which fixes all the ruby 2.7 warnings I could find. Please let me know if you find another one!
